cs.py

```diff
@@ -195,11 +124,7 @@ def _request(self, command, json=True, opcode_name='command', **kwargs):
             kw['data'] = kwargs
         response = getattr(requests, self.method)(self.endpoint, **kw)
         data = response.json()
-        key = '{0}response'.format(command.lower())
-        if key not in data and key in response_key_overrides:
-            key = response_key_overrides[key]
-            assert key in data, key
-
+        key = list(data.keys())[0]
```

Comment:
Hmm why not… I think I'd prefer `[key] = data.keys()` to be a little more strict.